doc/nrf/releases_and_maturity/known_issues.rst

```diff
@@ -60,6 +60,33 @@ The issues in this section are related to :ref:`protocols`.
 Amazon Sidewalk
 ===============
 
+.. rst-class:: v2-6-0
+
+KRKNWK-18465: BUS fault on FSK during the FACTORY_RESET call
+  The Bluetooth LE link is deinitialized.
+  However, a race condition might occur where an event added by the Bletooth LE link is not removed.
+  Acting on the event without a valid Bluetooth LE handle leads to dereference of ``NULL``.
+  To avoid the error, you need to check for ``NULL`` before the event is processed in the event queue.
+  The issue requires a fix in the Sidewalk stack.
+
+  **Workaround:** Perform a factory reset as the error is recoverable.
+
+.. rst-class:: v2-6-0
+
+KRKNWK-18511: Advertising fails to start (``err -12``) upon registration
+  The Sidewalk end device is trying to restart advertising, for example, on time sync lost.
+  This happens even when the device is connected through Bluetooth LE, leading to an error.
+  The Bluetooth LE connection management requires a fix in the Sidewalk stack.
+
+  **Workaround:** The issue fixes upon automatic restart of the Bluetooth LE advertising.
+
+.. rst-class:: v2-6-0
+
+KRKNWK-18499: Assertion fails during the stress test (pressing different buttons)
+  There is a corner case in the dynamic heap allocation, which leads to a double free error.
+
+  **Workaround:**  The device resets after assertion and functions normally.
```

Comment:
```suggestion
KRKNWK-18546: High current consumption for the Sidewalk application
  In the idle state, the release version of the Sidewalk samples consumes too much current (~240µA) due to enabled UART in the network core.

  **Affected platforms:** nRF5340 DK
```
This entry didn't stay here for too long - KRKNWK-18499 is fixed already:) adding a new one instead

Reply:
Thanks Anna. Any workaround?

Reply:
None for now